frontend/packages/console-app/src/components/cloud-shell/CloudShellTerminal.scss

```diff
@@ -0,0 +1,8 @@
+.odc-cloudshell-terminal {
+  &__container {
+    background-color: var(--pf-global--palette--black-1000);
```

Comment:
Does this even exist `black-1000` ?

Reply:
same is used in log component -- var(--pf-global--palette--black-1000);

Reply:
It seems like it does exist. But I wonder if we should just use `--pf-global--BackgroundColor--dark-100` instead.